test/test_functions

```diff
@@ -303,6 +305,17 @@ cvmfs_mount() {
   if [ "x$CVMFS_TEST_DEBUGLOG" != "x" ]; then
     sudo sh -c "echo \"CVMFS_DEBUGLOG=$CVMFS_TEST_DEBUGLOG\" >> /etc/cvmfs/default.local" || return 100
   fi
+
+  if running_on_osx; then
+    eval $(cvmfs_config showconfig | grep -e ^CVMFS_CONFIG_REPOSITORY=)
+    if [ "x$CVMFS_CONFIG_REPOSITORY" != "x" ]; then
```

Comment:
Keep in mind that `CVMFS_CONFIG_REPOSITORY` is optional.  But there is always something in `$repositories`